instructor/ui/instructor.js

```diff
@@ -163,6 +163,30 @@ $(function(){
     $("#id_has_nuclear_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
     $("#id_has_cytoplasmic_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
 
+    /**
+     * Western Blotting: band intensity
+     */
+    var sliders = $(".intensity_slider");
+    for(var i=0; i<sliders.length;i++){
```

Comment:
It may be cleaner to use `$(".intensity_slider").each(function($slider) { ...` instead, since you're not using the `i` variable anywhere else.